crates/js-sys/src/lib.rs

```diff
@@ -818,6 +818,12 @@ impl fmt::Debug for Boolean {
 // DataView
 #[wasm_bindgen]
 extern "C" {
+    /// An accessor to binary data stored inside an [ArrayBuffer]
+    ///
+    /// For statically typed views on the same buffer (which also offer tools for interacting with
+    /// Rust slices), see [Uint8Array], [Int64Array], [Float32Array] and so on. Beware that
+    /// `Uint8Array::new(&my_data_view)` *not* contain the full memory area; using the underlying
+    /// ArrayBuffer (`Uint8Array::new(&my_data_view.buffer())`) does.
```

Comment:
I'm not quite sure what this last sentence is supposed to be saying. `Uint8Array::new(&my_data_view)` doesn't work *at all*; it completely ignores the `DataView` and creates an empty `Uint8Array`. I'd phrase it something more along the lines of: 'A typed array cannot be created directly from a `DataView`; instead, it has to be created from its underlying `ArrayBuffer`: `Uint8Array::new(&my_data_view.buffer())`. Note that the resulting typed array will then be a view of the *entire* `ArrayBuffer`, even if the `DataView` was only a view of part of the `ArrayBuffer`.'